core/src/main/java/org/apache/carbondata/core/scan/scanner/AbstractBlockletScanner.java

```diff
@@ -46,6 +48,10 @@
 
   private AbstractScannedResult emptyResult;
 
+  private static int NUMBER_OF_ROWS_PER_PAGE = Integer.parseInt(CarbonProperties.getInstance()
```

Comment:
Actually it is not supposed to be configurable but as other places it is getting from carbon properties I am also getting like this. I will make it final.